src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/help/Get-AzureRmPowerBIEmbeddedCapacity.md

```diff
@@ -0,0 +1,133 @@
+---
+Download Help Link: None_Azure
+external help file: Microsoft.Azure.Commands.PowerBIDedicated.dll-Help.xml
+Help Version: 0.0.1.0
+Locale: en-US
+Module Guid: acace26c-1775-4100-85c0-20c4d71eaa22
+Module Name: AzureRM.PowerBIDedicated
+schema: 2.0.0
+---
+
+# Get-AzureRmPowerBIEmbeddedCapacity
+
+## SYNOPSIS
+Gets the details of an PowerBI Embedded Capacity.
+
+## SYNTAX
+
+```
+Get-AzureRmPowerBIEmbeddedCapacity [[-ResourceGroupName] <String>] [[-Name] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzureRmPowerBIEmbeddedCapacity cmdlet gets the details of an PowerBI Embedded Capacity.
+
+## EXAMPLES
+
+### Example 1: Get resource group capacities
+```
+PS C:\>Get-AzureRmPowerBIEmbeddedCapacity -ResourceGroupName "testRG"
+Sku                    : A1
+Tier                   : PBIE_Azure
+Administrator          : {{admin@microsoft.com}}
+State                  : Succeeded
+ProvisioningState      : Succeeded
+Id                     : /subscriptions/78e47976-009f-4d4a-a961-6046cdabf459/resourceGroups/testRG/providers/Microsoft.PowerBIDedicated/capacities/testcapacity
```

Comment:
@selasagi not sure if you mind this subscription id being publicly visible in the example